fix: Address unknown path format error in `wsl2`

snicmakino opened the pull request:
This change solves the problem [#371](https://github.com/serverless/serverless-python-requirements/issues/371) that is occurring in wsl2.
In order to make wsl2 work as well as linux, the wsl decision was changed so that it only applies to wsl1.

pgrzesik reviewed (changes requested):
Hello @snicmakino - thanks a lot for this, could you please address the formatting issues before merging?

snicmakino commented:
Hello @pgrzesik

> Hello @snicmakino - thanks a lot for this, could you please address the formatting issues before merging?

Thank you for pointing that out.
I fixed it and I confirmed that `prettier-check:updated` is fine.

pgrzesik reviewed (approved):
Thank you @snicmakino 👍

EnzoMolion commented:
Hi guys!

First, thanks for the amazing tool that is Serverless, it is much appreciated

Do you, by any chance, have an idea of when this fix will be shipped in a release?

Thanks

pgrzesik commented:
Hey @EnzoMolion - I was planning to do a release either today or early next week.

EnzoMolion commented:
Ok, thanks a lot

EnzoMolion commented:
Tested it today, it works like a charm, thanks @snicmakino and @pgrzesik
